lib/cogctl/actions/relays/delete.ex

```diff
@@ -0,0 +1,35 @@
+defmodule Cogctl.Actions.Relays.Delete do
+  use Cogctl.Action, "relays delete"
+
+  def option_spec do
+    []
+  end
+
+  def run(options, args, _config, %{token: nil}=endpoint) do
+    with_authentication(endpoint, &run(options, args, nil, &1))
+  end
+  def run(_options, [], _config, _endpoint), do: display_arguments_error
+  def run(_options, args, _config, endpoint) do
+    do_delete(endpoint, args)
+  end
+
+  defp do_delete(endpoint, relay_names) do
+    Enum.each(relay_names, fn relay_name ->
```

Comment:
I'll get rid of the case statement, but from your previous comment, you stated that we should be able to continue to delete even if a deletion fails - that makes sense to me. We should be able to display the error and move to the next relay to be deleted. For example, the following should happen if the first and last relays exist, but the second doesn't.
```
$ cogctl relays delete momomo mimimi mememe
Deleted momomo
ERROR: "Resource not found: The relay mimimi could not be deleted"
Deleted mememe
```